src/backend/TrafficCourts/Workflow.Service/Mappings/MappingProfile.cs

```diff
@@ -8,19 +8,20 @@ public class MessageContractToNoticeOfDisputeMappingProfile : Profile
     {
         public MessageContractToNoticeOfDisputeMappingProfile()
         {
-            CreateMap<SubmitNoticeOfDispute, NoticeOfDispute>()
-                .ForMember(dest => dest.DisputedCounts, opt => opt.MapFrom(src => src.DisputedCounts))
-                .ForMember(dest => dest.ViolationTicket, opt => opt.MapFrom(src => src.ViolationTicket))
-                // Parsing DateOfBirth to DateTime due to the DateOnly deserialization issues on oracle-data-api
-                .ForPath(dest => dest.ViolationTicket.Birthdate, 
-                        opt => { 
-                            opt.Condition(src => src.Source.ViolationTicket != null); 
-                            opt.MapFrom(src => src.ViolationTicket.Birthdate);
-                        });
-            CreateMap<Messaging.MessageContracts.DisputedCount, Models.DisputedCount>();
-            CreateMap<Messaging.MessageContracts.ViolationTicket, Models.ViolationTicket>();
-            CreateMap<Messaging.MessageContracts.TicketCount, Models.ViolationTicketCount>();
-            CreateMap<Messaging.MessageContracts.LegalRepresentation, Models.LegalRepresentation>();
+            CreateMap<SubmitNoticeOfDispute, Common.OpenAPIs.OracleDataApi.v1_0.Dispute>();
+
+            CreateMap<Messaging.MessageContracts.DisputedCount, Common.OpenAPIs.OracleDataApi.v1_0.DisputedCount>();
+            CreateMap<Messaging.MessageContracts.DisputedCount, Models.DisputedCount>();

```

Comment:
Not sure if we still need these Models.DisputedCount model classes. Maybe we can remove them as well in favour of the generated model in a subsequent PR.

Reply:
Removed unused models.